server/command_hooks.go

```diff
@@ -44,44 +56,77 @@ func (p *Plugin) emitStatusChange() {
 func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
 	configuration := p.getConfiguration()
 
-	if !strings.HasPrefix(args.Command, "/"+CommandTrigger) {
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-			Text:         fmt.Sprintf("Unknown command: " + args.Command),
-		}, nil
-	}
+	if strings.HasPrefix(args.Command, "/"+CommandTriggerPlugin) {
+
+		if strings.HasSuffix(args.Command, "true") {
+			if !configuration.disabled {
+				return &model.CommandResponse{
+					ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
+					Text:         "The demo plugin hooks are already enabled.",
+				}, nil
+			}
+
+			configuration.disabled = false
+			p.emitStatusChange()
 
-	if strings.HasSuffix(args.Command, "true") {
-		if !configuration.disabled {
 			return &model.CommandResponse{
 				ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-				Text:         "The demo plugin hooks are already enabled.",
+				Text:         "Enabled demo plugin hooks.",
 			}, nil
-		}
 
-		configuration.disabled = false
-		p.emitStatusChange()
+		} else if strings.HasSuffix(args.Command, "false") {
+			if configuration.disabled {
+				return &model.CommandResponse{
+					ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
+					Text:         "The demo plugin hooks are already disabled.",
+				}, nil
+			}
 
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-			Text:         "Enabled demo plugin hooks.",
-		}, nil
+			configuration.disabled = true
+			p.emitStatusChange()
 
-	} else if strings.HasSuffix(args.Command, "false") {
-		if configuration.disabled {
 			return &model.CommandResponse{
 				ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-				Text:         "The demo plugin hooks are already disabled.",
+				Text:         "Disabled demo plugin hooks.",
 			}, nil
 		}
-
-		configuration.disabled = true
-		p.emitStatusChange()
-
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-			Text:         "Disabled demo plugin hooks.",
-		}, nil
+	} else if strings.HasPrefix(args.Command, "/"+CommandTriggerEphemeral) {
+
+		URL := fmt.Sprintf("%s", *p.API.GetConfig().ServiceSettings.SiteURL)
+
+		post := &model.Post{
+			ChannelId: args.ChannelId,
+			UserId:    args.UserId,
+			Message:   "test ephemeral actions",
+			Props: model.StringInterface{
+				"attachments": []*model.SlackAttachment{
+					{
+						Actions: []*model.PostAction{
+							{
+								Integration: &model.PostActionIntegration{
+									Context: model.StringInterface{
```

Comment:
I would like (2/5) go get rid of `model.StringInterface` in this code base. What do you think about using `map[string]interface{}` instead?

Reply:
Using model.StringInterface looks cleaner to me personally.  I can make the switch to `map[string]interface{}` if you'd prefer though.

Reply:
I'm only 2/5. Let's go with `model.StringInterface`, if you prefer it.